Ginger/GingerCore/Actions/ActSikuli.cs

```diff
@@ -391,6 +391,11 @@ void RefreshActiveProcessesTitles()
                     lstWindows.Add(process);
                 }
             }
+            if (!string.IsNullOrEmpty(ProcessNameForSikuliOperation) &&
+                !ActiveProcessWindowsList.Contains(ProcessNameForSikuliOperation))
```

Comment:
If ActiveProcessWindowsList is already having the process name, will it show it as selected?

Reply:
Yes, that is handled by GingerCore.General.FillComboFromList(xActiveProcessesTitlesComboBox, actSikuli.ActiveProcessWindows); which is implemented in UI